### PR TITLE
Provide a fix for BarChart that is not rendered on a webpage with 2 nested master pages when ClientIDMode=AutoID (Resolve #211)

### LINKED ISSUE #211: BarChart is not rendered on a webpage when ClientIDMode is set to AutoID
I am getting an error with the Bar Chart after deploying a web page to Server 2012 R2.

 Unhandled exception at line 437, column 9 in http://localhost:61100/PartechTrans/ScriptResource.axd?d=z48y4oai1GgZ_a25brgQ9zD7rMfezY4sEp5Qnn37-rlZ89uRTV-Wsg-lM4_ZNzOWQ2VmVspM40ER72YqFszMf_2pESipSyouigdyVNX1wTvW4j2EqGJHpzdepIG3HgYn0&t=ffffffffcfeeeb28

0x800a138f - JavaScript runtime error: Unable to set property 'innerHTML' of undefined or null reference

Everything works perfectly on the DEV PC. 

From any web browser, accessing the page there is only a line across the page where the control should render. The data is being retrieved and is identifiable in View Source, but the control is not present.  

this line is where the error halts execution, when attempting to view the page on the test server within VS.

ln 437 =   this._parentDiv.innerHTML = svgContents;

I have an image attached.  

This is a fully deployed web site that is working, this was just adding a new page. The control is on an aspx page which has 2 master page levels. The first provides the navigation panel, and the top level parent master page provides the site header and other logo and login controls.

https://www.devexpress.com/Support/Center/Attachment/GetAttachmentFile/bd33d5f7-7082-11e6-80bf-00155d62480c

### PR DESCRIPTION
**Pull request by MikhailTymchukDX**
Container is found automatically using a ClientID property value passed from the server code.

### CODE REVIEW

File: AjaxControlToolkit/BarChart/BarChart.cs [hunk 1]
```diff
@@ -105,6 +105,11 @@ public class BarChart : ChartBase {
                     throw new Exception("Name is missing the BarChartSeries. Please provide a name in the BarChartSeries.");
             }
         }
+
+        protected override void DescribeComponent(ScriptComponentDescriptor descriptor) {
+            base.DescribeComponent(descriptor);
+            descriptor.AddElementProperty("container", ContainerID);
```
Comment (AlekseyMartynov) [inline AjaxControlToolkit/BarChart/BarChart.cs]: Use the `containerID` name on the client side too

Comment (MikhailTymchukDX) [inline AjaxControlToolkit/BarChart/BarChart.cs]: No, this is not an ID.
Please take note of the `Element` part in the `AddElementProperty` method - it resolves a DOM element on the client side automatically with the passed `ContainerID`.

**Comment by AlekseyMartynov** [comment]
LGTM then

**Pull request merged**